README.md

```diff
@@ -11,12 +11,17 @@ The "source of truth" for the event data format is in the form of
 See [proto/README.md](proto/README.md) for more
 information about the messages used to describe CloudEvents.
 
-## Stability warning
+## Stability
 
-The entire content of this repository should be regarded as highly
-unstable until this warning is removed. The schema may change, the
-CloudEvent "type" attribute values may change, the protobuf message
-may change. Even the core `cloud_event_type` annotation may change.
+The protobuf files in the [proto](proto/) directory of the default
+branch should be regarded as stable, with no further breaking
+changes. New events should not be added to the default branch until
```

Comment:
I don't think have or will use multiple branches (without intention for merging to the default branch) for new events in the future. Is this a new proposal? Do we plan for alpha beta event versioning for new events or go from branch to stable?

Describing this process a bit more might be useful - probably in a different README file.

Reply:
It's more leaving the door open to that in the future, rather than making a concrete proposal right now. I could take it out if you'd prefer.

Reply:
My opinion:

We should keep this top portion to the point. Viewers are reading this repo from Eventarc and GCF (GA) docs.

Suggestion, instead of a section, just have this paragraph with this formatting:

> **Stability**: The protobuf files in the [proto](proto/) directory should be regarded as stable with no further breaking changes. Other files such as generated JSON schema should not be regarded as stable.

We don't need to be extra verbose here. Developers want to see CloudEvents in this repo. We can add further details for maintainers and the exact meaning of stable in a different README (possibly link it).

We also have multiple GA products like Eventarc, GCF .NET, etc, that use these CloudEvents. It's a little scary seeing big disclaimers in the beginning of a repo.